packages/components/src/Avatar/Avatar.css

```diff
@@ -0,0 +1,44 @@
+:root {
+  --avatar-size: 2rem;
+  --avatar-border-size: 2px;
+}
+
+.avatar {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: var(--avatar-size);
+  height: var(--avatar-size);
+  border-radius: 50%;
+  color: var(--color-blue--dark);
+  font-size: var(--avatar-size);
+  -webkit-font-smoothing: antialiased;
+  background: no-repeat center center var(--color-greyBlue);
+  background-size: cover;
+}
+
+.large {
+  --avatar-size: 4.5rem;
+  --avatar-border-size: 4px;
+}
+
+.isDark {
+  color: var(--color-white);
+}
+
+.initials {
+  font-size: 0.5em;
```

Comment:
I don't know if it's possible but, it might be nice to tie the fixed sizes in here to a preset var. If not we should probably scale them based on [`--base-unit`](https://github.com/GetJobber/atlantis/blob/master/packages/design/src/spacing.css#L2).